codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2ConnectionHandlerBuilder.java

```diff
@@ -106,6 +105,8 @@
     private Boolean encoderEnforceMaxConcurrentStreams;
     private Boolean encoderIgnoreMaxHeaderListSize;
     private int initialHuffmanDecodeCapacity = DEFAULT_INITIAL_HUFFMAN_DECODE_CAPACITY;
+    private Http2PromisedRequestVerifier promisedRequestVerifier = ALWAYS_VERIFY;
+    private boolean autoAckSettings = true;
```

Comment:
`autoAckSettingsFrame`? If omitting `Frame` is a convention, please leave it as it is.

Reply:
+1 for rename. I'm not aware of any convention to omit `Frame`.